.codecov.yml

```diff
@@ -0,0 +1,18 @@
+coverage:
+    status:
+        # Make all cov less fatal initially
+        project:
+            default:
+                if_no_uploads: success
+                if_not_found: success
+        patch:
+            default:
+                if_no_uploads: success
+                if_not_found: success
+        changes:
+            default:
+                if_no_uploads: success
+                if_not_found: success
+
+comment:
+    layout: "diff"
```

Comment:
Do we really want comments at all? It seems quite annoying, I guess you can find the details going to the commit status link anyway?